src/compas_timber/_fabrication/step_joint.py

```diff
@@ -75,7 +75,7 @@ def __init__(
         tenon=False,
         tenon_width=40.0,
         tenon_height=40.0,
-        **kwargs
+        **kwargs,
```

Comment:
There is this issue, where the formatter adds a trailing comma after kwargs in __init__ on long lines, causing an 'unexpected token' error in IronPython. I guess to avoid that we can add `# fmt: off` as a comment directly above the __init__ method, when line breaking happens to prevent from reformatting each time the formatter runs."

Reply:
@chenkasirer what is the best soluiton here?

Reply:
what panos said ;)

Reply:
we should do this on another PR

Reply:
why? isn't this breaking IronPython?

Reply:
Guys that line already exists in main! @obucklin make sure to pull from main. I added it in one of the PR's I merged(double_cut). The issue was only on these two modules.